### PR TITLE
Add Autogiro references

### PR DESCRIPTION
**Pull request by duncanbarrigan**
This branch adds references to Autogiro and Sweden to the English-language European splash pages. 

@Ken-Lo and others, do you know of other references I've missed? Please also check I haven't broken anything!

A couple of other points:
- We will need to update the all the pages again as we add new schemes like the US and Australia (obviously at some point we will have to change the language rather than simply adding them). Once we finalise this set I'll make a list of the points to change so that it's easier next time
- In the long run we'll need to be careful about these updates for the non-English sites too ( @octaveauger , @jutta-frieden , @AlfonsoSB , @Jennabrown )

### CODE REVIEW

File: app/pages/pro/pro.en.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ export default class ProEn extends React.Component {
                     Designed specifically for larger enterprises,
                     GoCardless Pro combines the simplicity of our original
                     GoCardless product with complete control over payments and customer experience.
-                    It also allows you to collect UK and SEPA Direct Debit via one simple integration.
+                    It also allows you to collect Direct Debits from the UK, Sweden and the Eurozone via one simple integration.
```
Comment (Ken-Lo) [inline app/pages/pro/pro.en.js]: [We normally write](https://brand.gocardless.com/#direct-debit) "Direct Debit payments" as opposed to pluralising as "Direct Debits"

**Comment by Ken-Lo** [comment]
@duncanbarrigan Looks good to me with minor comment above! :+1:

@octaveauger @jamesshedden Any thoughts on merging gocardless.com/europe/ into gocardless.com/en-eu/? Otherwise, I feel /europe/ is going to continue lagging behind. On the en-EU website, we have much more info aimed at non-UK users (e.g. Pro Dashboard tutorial). Would also be good to link to our Bacs, SEPA & Autogiro guides from one page.

**Comment by octaveauger** [comment]
Yes I think it'd be better on en-eu: why limit it to one page when we could
have a whole site with relevant doc that's already mostly there. The main
thing would be to keep the way we record leads in the DB, as this is
helpful for future country leads.

Btw we use "Direct Debits" everywhere on the UK site ;)

Octave Auger
co-Directeur France
+33 (0) 9 87 67 94 20 | +44 (0) 7450 245 225
www.gocardless.fr
Ils parlent de GoCardless: Le Figaro
<http://www.lefigaro.fr/societes/2015/04/12/20005-20150412ARTFIG00171-gocardless-pret-a-bousculer-les-banques-francaises.php>
| Les Echos
<http://www.lesechos.fr/finance-marches/banque-assurances/0213743712-le-prelevement-bancaire-seduit-de-nouveaux-acteurs-1110875.php>
| Challenges
<http://www.challenges.fr/finance-et-marche/20150413.CHA4873/ces-start-up-qui-font-valser-les-modeles-de-la-city-de-londres.html>
| La Tribune
<http://www.latribune.fr/entreprises-finance/banques-finance/gocardless-ou-quand-la-fintech-londonienne-debarque-en-france-469023.html>
| Journal du net
<http://www.journaldunet.com/web-tech/start-up/fintech-gocardless-lancement-france.shtml>
|
BFM TV
<http://bfmbusiness.bfmtv.com/mediaplayer/video/start-up-co-gocardless-la-fintech-qui-veut-revolutionner-les-prelevements-bancaires-pour-les-entreprises-2204-508599.html>

File: app/pages/pro/pro.en.js (hunk 1 above)
Comment (duncanbarrigan) [inline app/pages/pro/pro.en.js]: Will make the edit now.

**Comment by duncanbarrigan** [comment]
Any other comments on this one or shall we merge it? 
I'll make a list of the points to change tomorrow.

**Comment by Ken-Lo** [comment]
@duncanbarrigan Nothing more from me, we can deal with the /europe/ to /en-eu/ merger in a separate PR.
Great work! :+1:

**Pull request merged**